crates/bevy_ui/src/lib.rs

```diff
@@ -46,14 +46,17 @@ use widget::UiImageSize;
 ///
 /// This includes the most common types in this crate, re-exported for your convenience.
 pub mod prelude {
+    #[allow(deprecated)]
+    #[doc(hidden)]
+    pub use crate::widget::TextBundle;
     #[doc(hidden)]
     pub use {
         crate::{
             geometry::*,
             node_bundles::*,
             ui_material::*,
             ui_node::*,
-            widget::{Button, Label, Text, UiTextReader, UiTextWriter},
+            widget::{Button, Label, Text, TextUiReader, TextUiWriter},
```

Comment:
The last three should be `"bevy_text"` feature-gated?

Reply:
Hm, `bevy_ui` doesn't have a `bevy_text` feature flag, guess I'll need to add that

Reply:
`bevy_ui` feature always imports `bevy_text`
It's a bit confusing that `bevy_text` is marked as an optional dependency, so I'll remove that.

Reply:
Hm, that didn't work 🤔 
Do dependencies act as features if marked as optional?
I thought we need to add a feature flag that then does `dep:dependency_crate`

Reply:
I'm not sure I don't understand rust feature gating that well. `bevy_ui` is meant to be able to run exclusively `bevy_text` but that doesn't seem possible atm.

Reply:
> Do dependencies act as features if marked as optional?

Yes.